### ISSUE TITLE
Container fails to start due to missing "static" folder 

### ISSUE THREAD

**Issue by daniel-gallagher** [Netflix/dispatch#60]
When running `docker-compose up` after install.sh script finishes, the following error appears: 

```
dispatch-scheduler_1  | Traceback (most recent call last):
dispatch-scheduler_1  |   File "/usr/local/bin/dispatch", line 8, in <module>
dispatch-scheduler_1  |     sys.exit(entrypoint())
dispatch-scheduler_1  |   File "/usr/local/lib/python3.8/site-packages/dispatch/cli.py", line 766, in entrypoint
dispatch-scheduler_1  |     dispatch_cli()
dispatch-scheduler_1  |   File "/usr/local/lib/python3.8/site-packages/click/core.py", line 764, in __call__
dispatch-scheduler_1  |     return self.main(*args, **kwargs)
dispatch-scheduler_1  |   File "/usr/local/lib/python3.8/site-packages/click/core.py", line 717, in main
dispatch-scheduler_1  |     rv = self.invoke(ctx)
dispatch-scheduler_1  |   File "/usr/local/lib/python3.8/site-packages/click/core.py", line 1137, in invoke
dispatch-scheduler_1  |     return _process_result(sub_ctx.command.invoke(sub_ctx))
dispatch-scheduler_1  |   File "/usr/local/lib/python3.8/site-packages/click/core.py", line 1137, in invoke
dispatch-scheduler_1  |     return _process_result(sub_ctx.command.invoke(sub_ctx))
dispatch-scheduler_1  |   File "/usr/local/lib/python3.8/site-packages/click/core.py", line 956, in invoke
dispatch-scheduler_1  |     return ctx.invoke(self.callback, **ctx.params)
dispatch-scheduler_1  |   File "/usr/local/lib/python3.8/site-packages/click/core.py", line 555, in invoke
dispatch-scheduler_1  |     return callback(*args, **kwargs)
dispatch-scheduler_1  |   File "/usr/local/lib/python3.8/site-packages/uvicorn/main.py", line 331, in main
dispatch-scheduler_1  |     run(**kwargs)
dispatch-scheduler_1  |   File "/usr/local/lib/python3.8/site-packages/uvicorn/main.py", line 354, in run
dispatch-scheduler_1  |     server.run()
dispatch-scheduler_1  |   File "/usr/local/lib/python3.8/site-packages/uvicorn/main.py", line 382, in run
dispatch-scheduler_1  |     loop.run_until_complete(self.serve(sockets=sockets))
dispatch-scheduler_1  |   File "uvloop/loop.pyx", line 1456, in uvloop.loop.Loop.run_until_complete
dispatch-scheduler_1  |   File "/usr/local/lib/python3.8/site-packages/uvicorn/main.py", line 389, in serve
dispatch-scheduler_1  |     config.load()
dispatch-scheduler_1  |   File "/usr/local/lib/python3.8/site-packages/uvicorn/config.py", line 288, in load
dispatch-scheduler_1  |     self.loaded_app = import_from_string(self.app)
dispatch-scheduler_1  |   File "/usr/local/lib/python3.8/site-packages/uvicorn/importer.py", line 20, in import_from_string
dispatch-scheduler_1  |     module = importlib.import_module(module_str)
dispatch-scheduler_1  |   File "/usr/local/lib/python3.8/importlib/__init__.py", line 127, in import_module
dispatch-scheduler_1  |     return _bootstrap._gcd_import(name[level:], package, level)
dispatch-scheduler_1  |   File "<frozen importlib._bootstrap>", line 1014, in _gcd_import
dispatch-scheduler_1  |   File "<frozen importlib._bootstrap>", line 991, in _find_and_load
dispatch-scheduler_1  |   File "<frozen importlib._bootstrap>", line 975, in _find_and_load_unlocked
dispatch-scheduler_1  |   File "<frozen importlib._bootstrap>", line 671, in _load_unlocked
dispatch-scheduler_1  |   File "<frozen importlib._bootstrap_external>", line 783, in exec_module
dispatch-scheduler_1  |   File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
dispatch-scheduler_1  |   File "/usr/local/lib/python3.8/site-packages/dispatch/main.py", line 32, in <module>
dispatch-scheduler_1  |     frontend.mount("/", StaticFiles(directory=STATIC_DIR), name="app")
dispatch-scheduler_1  |   File "/usr/local/lib/python3.8/site-packages/starlette/staticfiles.py", line 55, in __init__
dispatch-scheduler_1  |     raise RuntimeError(f"Directory '{directory}' does not exist")
dispatch-scheduler_1  | RuntimeError: Directory '/usr/local/lib/python3.8/site-packages/dispatch/static/dispatch/dist' does not exist
web_1                 | Traceback (most recent call last):
web_1                 |   File "/usr/local/bin/dispatch", line 8, in <module>
web_1                 |     sys.exit(entrypoint())
web_1                 |   File "/usr/local/lib/python3.8/site-packages/dispatch/cli.py", line 766, in entrypoint
web_1                 |     dispatch_cli()
web_1                 |   File "/usr/local/lib/python3.8/site-packages/click/core.py", line 764, in __call__
web_1                 |     return self.main(*args, **kwargs)
web_1                 |   File "/usr/local/lib/python3.8/site-packages/click/core.py", line 717, in main
web_1                 |     rv = self.invoke(ctx)
web_1                 |   File "/usr/local/lib/python3.8/site-packages/click/core.py", line 1137, in invoke
web_1                 |     return _process_result(sub_ctx.command.invoke(sub_ctx))
web_1                 |   File "/usr/local/lib/python3.8/site-packages/click/core.py", line 1137, in invoke
web_1                 |     return _process_result(sub_ctx.command.invoke(sub_ctx))
web_1                 |   File "/usr/local/lib/python3.8/site-packages/click/core.py", line 956, in invoke
web_1                 |     return ctx.invoke(self.callback, **ctx.params)
web_1                 |   File "/usr/local/lib/python3.8/site-packages/click/core.py", line 555, in invoke
web_1                 |     return callback(*args, **kwargs)
web_1                 |   File "/usr/local/lib/python3.8/site-packages/uvicorn/main.py", line 331, in main
web_1                 |     run(**kwargs)
web_1                 |   File "/usr/local/lib/python3.8/site-packages/uvicorn/main.py", line 354, in run
web_1                 |     server.run()
web_1                 |   File "/usr/local/lib/python3.8/site-packages/uvicorn/main.py", line 382, in run
web_1                 |     loop.run_until_complete(self.serve(sockets=sockets))
web_1                 |   File "uvloop/loop.pyx", line 1456, in uvloop.loop.Loop.run_until_complete
web_1                 |   File "/usr/local/lib/python3.8/site-packages/uvicorn/main.py", line 389, in serve
web_1                 |     config.load()
web_1                 |   File "/usr/local/lib/python3.8/site-packages/uvicorn/config.py", line 288, in load
web_1                 |     self.loaded_app = import_from_string(self.app)
web_1                 |   File "/usr/local/lib/python3.8/site-packages/uvicorn/importer.py", line 20, in import_from_string
web_1                 |     module = importlib.import_module(module_str)
web_1                 |   File "/usr/local/lib/python3.8/importlib/__init__.py", line 127, in import_module
web_1                 |     return _bootstrap._gcd_import(name[level:], package, level)
web_1                 |   File "<frozen importlib._bootstrap>", line 1014, in _gcd_import
web_1                 |   File "<frozen importlib._bootstrap>", line 991, in _find_and_load
web_1                 |   File "<frozen importlib._bootstrap>", line 975, in _find_and_load_unlocked
web_1                 |   File "<frozen importlib._bootstrap>", line 671, in _load_unlocked
web_1                 |   File "<frozen importlib._bootstrap_external>", line 783, in exec_module
web_1                 |   File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
web_1                 |   File "/usr/local/lib/python3.8/site-packages/dispatch/main.py", line 32, in <module>
web_1                 |     frontend.mount("/", StaticFiles(directory=STATIC_DIR), name="app")
web_1                 |   File "/usr/local/lib/python3.8/site-packages/starlette/staticfiles.py", line 55, in __init__
web_1                 |     raise RuntimeError(f"Directory '{directory}' does not exist")
web_1                 | RuntimeError: Directory '/usr/local/lib/python3.8/site-packages/dispatch/static/dispatch/dist' does not exist
```

The cause seems to be that the `src/dispatch/static` folder is missing in the built docker image.

**Comment by lastlee** [comment]
This seems like an issue with the setup.py configuration where the static files are not included when you run the `pip install` -- not sure where the static files ultimately end up, but the app is certainly looking for them in site-packages.

**Comment by daniel-gallagher** [comment]
I believe this can be fixed by creating a MANIFEST.in file that contains 
```
recursive-include src/dispatch *
```

This should also fix #59

**Comment by daniel-gallagher** [comment]
This is now resolved after the merge of #62.
